src/relay/transforms/pattern_utils.h

```diff
@@ -700,6 +700,13 @@ Expr StopFusion(Expr data);
 
 Expr CastHint(Expr data, DataType dtype);
 
+inline Expr Concat(Expr x, int axis = 0) {
+  static const Op& op = Op::Get("concatenate");
+  auto attrs = make_object<ConcatenateAttrs>();
+  attrs->axis = axis;
+  return Call(op, {x}, Attrs(attrs), {});
+}
```

Comment:
Maybe just use the version here instead of duplicating? https://github.com/apache/tvm/blob/2915349458619deb5bd9b03670b33205938a8c02/src/relay/op/make_op.h#L45

Reply:
Done. Thank you!